app/assets/stylesheets/base/_buttons.scss

```diff
@@ -0,0 +1,30 @@
+.button {
+  width: 100%;
+  padding: 10px 0;
+  transition: $basic-transition;
+  border-radius: 55px;
```

Comment:
Should we use `rem` instead of `px` project?
Because most of our production is using this 🙌

Feel free to use this function [here](https://github.com/nimblehq/rails-templates/blob/develop/.template/variants/web/app/assets/stylesheets/functions/_sizing.scss)

Reply:
Done 😎 Updated in https://github.com/aetheryna/google-scraper-ruby/pull/32/commits/6e5051013e49fbd4fc6fe209042953850e62b12d